srcpkgs/cglm/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'cglm'
 pkgname=cglm
-version=0.9.2
+version=0.9.4
 revision=1
 build_style=meson
 short_desc="Highly Optimized Graphics Math (glm) for C"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Aaditya Aryal <aryalaadi123@gmail.com>"
```

Comment:
Please drop the maintainer change; we like to see a longer history of package management before people put their names on packages.

Reply:
okay